config/crd/bases/postgres-operator.crunchydata.com_postgresclusters.yaml

```diff
@@ -8771,6 +8771,26 @@ spec:
                           may also be set using the RELATED_IMAGE_PGADMIN environment
                           variable. More info: https://kubernetes.io/docs/concepts/containers/images'
                         type: string
+                      ldapBindPassword:
+                        description: The Secret containing the pgAdmin value for the
+                          LDAP_BIND_PASSWORD configuration setting which is used for
+                          the pgAdmin login user when LDAP authentication is enabled.
+                        properties:
+                          key:
+                            description: The key of the secret to select from.  Must
+                              be a valid secret key.
+                            type: string
+                          name:
+                            description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                              TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
📝 We have to run `hack/create-todo-patch.sh` to remove these `TODO`. It looks like we haven't done it in a while, so it will pull in more than just this one occurrence. It's a little fiddly; I think it has to run between two `make generate` or something?

🏕️ If you want to try it now, go for it, otherwise, I can take care of it in a separate PR.

Reply:
I gave it a shot, but I think I have the wrong version of `yq`. I'll leave that to you, if you don't mind :slightly_smiling_face: